src/Util/Temporal.php

```diff
@@ -305,13 +305,14 @@ public static function getDateTimeField(
 	 * Results relative to current timezone.
 	 * Limited to range of timestamps.
 	 *
-	 * @param string $posted_date MySQL-formatted date string (YYYY-MM-DD HH:MM:SS)
-	 * @param string $format (optional) Parsed with sprintf()
+	 * @param string $posted_date       MySQL-formatted date string (YYYY-MM-DD HH:MM:SS)
+	 * @param string $format            (optional) Parsed with sprintf()
+	 * @param bool   $compare_date_time Compare date (false) or datetime (true). "true" is default.
 	 *    <tt>%1$d %2$s ago</tt>, e.g. 22 hours ago, 1 minute ago
 	 *
 	 * @return string with relative date
 	 */
-	public static function getRelativeDate(string $posted_date = null, string $format = null): string
+	public static function getRelativeDate(string $posted_date = null, string $format = null, bool $compare_date_time = true): string
```

Comment:
Please simplify the parameter name.
```suggestion
	public static function getRelativeDate(string $posted_date = null, string $format = null, bool $compare_time = true): string
```

Reply:
done